numpy/lib/type_check.py

```diff
@@ -343,7 +343,7 @@ def nan_to_num(x, copy=True):
 
     Parameters
     ----------
-    x : array_like
+    x : scalar or array_like
```

Comment:
Scalars are effectively already considered array_like, I think?

Reply:
I found 13 occurrences of `scalar or array_like` in the docs, so I decided to place it here as well to make it clearer. I can remove it if you think it is redundant though.

Reply:
Sounds like a fair argument